net.crossfireatlas.crossfire.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.crossfireatlas.crossfire</id>
+	<name>Crossfire RPG</name>
+	<summary>Roguelike multi-user RPG</summary>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>GPL-2.0</project_license>
+	<!-- Description taken from https://crossfire.real-time.com -->
+	<description>
+                <p>"Crossfire is a multi-player graphical arcade and adventure game originally made for the X Windows systems environment. It has certain flavours from other games, especially gauntlet and the rogue-like games (Nethack, Moria, Angband, and Ragnarok). Any number of players can move around in their own window, finding and using items and battling monsters. They can choose to cooperate or compete in the same world.''</p>
+                <p>In the years of development that have followed, Crossfire has grown to encompass over 150 monsters, about 3000 maps to explore, an elaborate magic system, over 15 character types, a system of skills, and many, many artifacts and treasures.</p>
+	</description>
+	<developer_name>Crossfire developers, see http://crossfire.real-time.com/</developer_name>
+	<url type="homepage">https://crossfire.real-time.com/</url>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://raw.githubusercontent.com/flathub/net.crossfireatlas.crossfire/master/screenshot1.png</image>
```

Comment:
this is not a screenshot, this is a banner / logo.

Reply:
OK, yes, that is correct. I removed the banner and left the screenshot (one).